test/integration/swap/singleChain.test.ts

```diff
@@ -425,6 +451,14 @@ function testFee(chain: Chain) {
 describe('Swap Single Chain Flow', function () {
   this.timeout(TEST_TIMEOUT)
 
+  describeExternal('Claim works after expiration', () => {
```

Comment:
Why is this in a separate section like this? Instead of another test in `testSwap`? 
This test should pass for every swap situation

Reply:
This could cause test failing sometimes due to the nature of how tests are done for Near, Solana and Terra (testnets not local nodes)

Reply:
in that case `testClaimSucceedAfterExpiration` should be called in each appropriate section
like `Bitcoin - Node`

This way, it's properly covered every scenario but still can pick an choose which chain to test